week-2/F-logical-operators/exercise2.js

```diff
@@ -5,8 +5,18 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
-
+function isNegative(number) {
+  return number < 0;
+}
+function isBetween5and10(number) {
+  return number >= 5 && number <= 10;
+}
+function isLessThan8Characters(name) {
+  return "daniel".length < 8;
```

Comment:
The main point of writing functions is that they can be used with different parameters. You need to write the function independently of any hard-coded values for the parameters (i.e. use the `name` parameter, not the string "daniel"). As it is, this function will always return `true` for any input. So `isLessThan8Characters("hippopotamus")` will be true, for example, which we don't want.